optaplanner-wb-webapp/pom.xml

```diff
@@ -589,7 +589,7 @@
     <!-- KIE Workbench Playground Repository -->
     <dependency>
       <groupId>org.kie.workbench.playground</groupId>
-      <artifactId>playground-parent</artifactId>
+      <artifactId>drools-playground-parent</artifactId>
```

Comment:
Bad name because it contains both drools and optaplanner examples.
Let's rename playground-parent to kie-playground-parent and keep it in the existing repository to minimize the build time.